### DISCUSSION TITLE
Mística Tokens | Structure Definition

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#800] [Proposals]
Abro esta discussion para listar qué elementos deberían tokenizarse en Mística

- [x] Colors variables (Light & Dark)
- [x] Colors constants (Light & Dark)
- [x] Texts variables
- [x] Texts constants
- [x] Border radius
- [ ] Shadows
- [ ] Animation curves
- [ ] Animation timings
- [ ] Animation delays
- [x] Media queries
- [ ] Opacity (?)
- [ ] [Z-Index](https://github.com/Telefonica/mistica-design/discussions/800#discussioncomment-3745871) (?)
- [ ] [Spacing](https://github.com/Telefonica/mistica-design/discussions/800#discussioncomment-3745288) (?)
- [ ] Blur (?)

**Comment by yceballost** [comment]
### Spacing 
No usaría los tokens de spacing para diseñar ritmos verticales pero si que creo que pueden ser interesantes para definir por ejemplo los padding entre diferentes breakpoints para mobile (small phones / large phones)

**Comment by yceballost** [comment]
### Z-Index
Me parece super interesante tener el Z-index controlado porque en proyectos grandes es algo que fácilmente se descontrola y terminas volviendote loco para manejar.
https://www.smashingmagazine.com/2021/02/css-z-index-large-projects/

**Reply by aweell** [reply]
Dejo por aquí una referencia que me parece bastante acertada desde el punto de vista semántico:

https://www.lightningdesignsystem.com/design-tokens/#category-z-index
